web/packages/test/config/launch-config.toml

```diff
@@ -14,14 +14,14 @@ default_args = [
     "--pruning", "archive",
 ]
 
-    [relaychain.genesis.runtime]
-    session_length_in_blocks = 20
+    # [relaychain.genesis.runtime]
+    # session_length_in_blocks = 20
 
-        [relaychain.genesis.runtime.runtime_genesis_config.configuration.config]
-        max_validators_per_core = 2
-        needed_approvals = 1
-        validation_upgrade_cooldown = 2
-        validation_upgrade_delay = 2
+    #     [relaychain.genesis.runtime.runtime_genesis_config.configuration.config]
+    #     max_validators_per_core = 2
+    #     needed_approvals = 1
+    #     validation_upgrade_cooldown = 2
+    #     validation_upgrade_delay = 2
```

Comment:
Did this config cause issues with the smoke tests?

Reply:
Nope, just for quick smoke tests more agile. I remember the previous change for the `session_length` is for debugging beefy commitment issues which may not be necessary now.